### PR TITLE
fix inheritence for resources

### PR DESCRIPTION
**Pull request by ibesso-rh**
##### Short description:
Correcting inheritence to certain resources

##### What this PR does / why we need it:
tests requires this to fetch those resources, which is currently not possible.

##### Special notes for reviewer:
release.sh was fixed by pre-commit

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -52,4 +52,4 @@ git commit -a -m"Update changelog for version $VERSION"
 git push -f origin $TARGET_BRANCH
 
 git pull origin $TARGET_BRANCH
-git checkout $MASTER_BRANCH
\ No newline at end of file
+git checkout $MASTER_BRANCH
```
Comment (rnetser) [inline release.sh:55]: please rebase

**Review by rnetser — CHANGES_REQUESTED** [review]

File: release.sh (hunk 1 above)
Comment (ibesso-rh) [reply release.sh]: done

**Comment by ibesso-rh** [comment]
@myakove , @rnetser , for some reason, the checklist still show "Pending - needs aprove from maintainers". could you please re-approve or check what was wrong?

**Comment by myakove** [comment]
@ofirnash1 please review

**Pull request merged**